services/web/server/src/simcore_service_webserver/projects/models.py

```diff
@@ -97,6 +98,15 @@ class Config:
         orm_mode = True
 
 
+class ProjectPatchExtended(ProjectPatch):
+    # Only used internally
+    trashed_at: datetime | None = None
```

Comment:
TIP: when you will be working on the DB layer, please add it also here: `ProjectDB(BaseModel)` this is a pure Project DB model one to one with projects table